### PR TITLE
fix: Atlas lineage multiple parents

### PR DESCRIPTION
**Pull request by mgorsk1**
### Summary of Changes

Adjusts AtlasProxy lineage rendering after changes to UI

### Tests

n/a

### Documentation

n/a

### CheckList
Make sure you have checked **all** steps below to ensure a timely review.
- [x] PR title addresses the issue accurately and concisely. Example: "Updates the version of Flask to v1.0.2"
    - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
- [x] PR includes a summary of changes.
- [x] PR adds unit tests, updates existing unit tests, __OR__ documents why no test additions or modifications are needed.
- [x] In case of new functionality, my PR adds documentation that describes how to use it.
    - All the public functions and the classes in the PR contain docstrings that explain what it does

### CODE REVIEW

**Comment by mgorsk1** [comment]
cc @verdan @bbsbb

File: metadata/metadata_service/proxy/atlas_proxy.py [hunk 1]
```diff
@@ -1506,7 +1506,7 @@ def _get_lineage_graph(self, lineage: Dict, entity_type: str, key_class: Any) ->
         return dict(graph)
 
     def _serialize_lineage_item(self, edge: Tuple[str, str], direction: str, key_class: Any,
-                                graph: Dict, root_node: str, parent_nodes: Dict[str, Set[str]]) -> LineageItem:
+                                graph: Dict, root_node: str, parent_nodes: Dict[str, Set[str]]) -> List[LineageItem]:
         """
         Renders LineageItem object.
```
Comment (verdan) [inline metadata/metadata_service/proxy/atlas_proxy.py:1511]:
```suggestion
        Serializes a list of lineage items
```

File: metadata/metadata_service/proxy/atlas_proxy.py [hunk 2]
```diff
@@ -1518,6 +1518,8 @@ def _serialize_lineage_item(self, edge: Tuple[str, str], direction: str, key_cla
         :param parent_nodes: Dict of keys (nodes) with set of keys (parents).
         :return: Serialized LineageItem object.
```
Comment (verdan) [inline metadata/metadata_service/proxy/atlas_proxy.py:1519]:
```suggestion
        :return: Serialized LineageItem list.
```

**Review by verdan — APPROVED** [review]
a couple of nits. Rest looks good to me.

**Comment by mgorsk1** [comment]
Thanks @verdan !

**Pull request merged**